docker/Dockerfile.cuda

```diff
@@ -0,0 +1,25 @@
+FROM nvidia/cuda:11.8.0-cudnn8-runtime-ubuntu22.04
+
+ARG DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt-get install -y \
+	ffmpeg \
+	nvidia-cudnn \
+	python3 \
+	python3-pip \
+	python3-tk \
+	python-is-python3
+
+WORKDIR /app
```

Comment:
Same like the other Dockerfile - should be a production only container

Reply:
fixed